code/ui/manager/src/components/panel/panel.tsx

```diff
@@ -29,22 +25,18 @@ class SafeTab extends Component<SafeTabProps, { hasError: boolean }> {
 
   render() {
     const { hasError } = this.state;
-    const { children, title, id } = this.props;
+    const { children } = this.props;
     if (hasError) {
       return <h1>Something went wrong.</h1>;
     }
-    return (
-      <SafeTabContent id={id} title={title}>
-        {children}
-      </SafeTabContent>
-    );
+    return children;
```

Comment:
Why is `SafeTabContent` not needed anymore?

Reply:
It wasn't doing anything at all..
Whatever it did in the past got refactored out I guess.